blocks/homesection/homesection.css

```diff
@@ -0,0 +1,5 @@
+.homesection {
+    background-color: lightgray;
+    color: black;
+    padding: 30px;
+}
```

Comment:
Are you referring to heroBanner?

Reply:
no. i have created it based on the video shared in the training link.